centos-vm-libvirt/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/pod_utils/systemd-libvirt:latest
+FROM quay.io/pod_utils/systemd-libvirt:1.0.3
```

Comment:
commit message should mention this change

Reply:
+1